tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -1375,3 +1387,9 @@ When(
     )
   }
 )
+
+Given('the administrator has set the default folder for received shares to {string}', function(
+  folder
+) {
+  return runOcc([`config:system:set share_folder --value=${folder}`])
```

Comment:
What will happen when testing on OCIS?

I don't see this actually running in a scenario in the CI against OCIS.

For example, the PHP code in core checks if `isTestingOnOcis` and the requested share folder is `Shares` then do nothing and return success.